shell/artifacts/Restaurants/source/RestaurantDetail.js

```diff
@@ -97,11 +200,15 @@ ${styles}
       };
       let detail;
       if (state.detail) {
+        var url = state.detail.website.replace(/^(?:https?:\/\/)?(?:www\.)?/i, "").split('/')[0];
```

Comment:
this would error out if state.detail.website isn't defined. not sure whether it defaults to an empty string, but better don't rely on it.